### DISCUSSION TITLE
Migrating from Zod to Valibot

### DISCUSSION THREAD

**Discussion by cr101** [epicweb-dev/epic-stack#602] [Q&A] [answered]
Has anyone used or recommend [Valibot](https://mwskwong.com/blog/migrating-from-zod-to-valibot-a-comparative-experience)?

**Comment by kentcdodds** [comment, marked as answer]
I personally prefer Zod's syntax. The size difference in Valibot is not significant enough for me to switch (especially in a real-world app where you're going to likely be using many of the validators anyway). Also, I don't know whether conform supports Valibot as well. I'm just not interested in changing from Zod.
